adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostUtils.m

```diff
@@ -72,13 +72,13 @@ void GADMAdapterChartboostMapTableSetObjectForKey(NSMapTable *_Nonnull mapTable,
       [string stringByTrimmingCharactersInSet:NSCharacterSet.whitespaceCharacterSet];
   if (!adLocation.length) {
     NSLog(@"Missing or Invalid Chartboost location. Using Chartboost's default location.");
-    return [CBLocationDefault copy];
+    return @"Default";
   }
   return adLocation;
 }
 
 CHBMediation *_Nonnull GADMAdapterChartboostMediation(void) {
-  return [[CHBMediation alloc] initWithType:CBMediationAdMob
+  return [[CHBMediation alloc] initWithName:@"AdMob"
```

Comment:
Is there a constant from Chartboost SDK that can be used rather than hardcoding it?

Reply:
We removed Mediation Constans from SDK